spec/views/accounts/_identity_verification.html.erb_spec.rb

```diff
@@ -59,11 +71,27 @@
         strip_tags(t('account.index.verification.finish_verifying_html', sp_name: ipp_sp_name)),
       )
     end
+
+    context 'with vtr values' do
+      let(:vtr) { ['C2'] }
+      let(:acr_values) { nil }
+
+      it 'references initiating sp with prompt to finish verifying their identity' do
```

Comment:
Does 'references initiating SP in prompt instructing user to complete identity verification' read better?